numpy/core/setup.py

```diff
@@ -14,7 +14,7 @@
     uses_accelerate_framework, get_sgemv_fix
     )
 from numpy.compat import npy_load_module
-from setup_common import *
+from setup_common import *  # noqa: F403
```

Comment:
For the record, I had difficulties making this into either a relative or absolute import. I can't recall exactly, but I think I tried `from .setup_common import *` and `from numpy.core.setup_common import *`, but both bombed Travis CI tests. If anyone knows why, I'd be interested.